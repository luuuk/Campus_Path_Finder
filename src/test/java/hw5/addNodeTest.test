# This test is for the addNode method

# Create a new Graph, g
CreateGraph g

# Add Node n1 to g
AddNode g n1