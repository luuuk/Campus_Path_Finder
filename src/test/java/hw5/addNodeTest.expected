# This test is for the addNode method

# Create a new Graph, g
created graph g

# Add Node n1 to g
added node n1 to g